documentation/pom.xml

```diff
@@ -34,102 +34,98 @@
-            <skip-front-matter>true</skip-front-matter>
-            <quickstart-clone-url>${extension-url}/tree/master/quickstart</quickstart-clone-url>
-            <reference-conf-url>${datastax-driver-documentation-url}/core/configuration/reference/</reference-conf-url>
-            <fruits-html-url>${extension-url}/tree/master/quickstart/src/main/resources/META-INF/resources/fruits.html</fruits-html-url>
-            <mapper-docs-url>${datastax-driver-documentation-url}/mapper</mapper-docs-url>
-          </attributes>
-        </configuration>
-        <executions>
-          <execution>
-            <id>output-html</id>
-            <phase>prepare-package</phase>
-            <goals>
-              <goal>process-asciidoc</goal>
-            </goals>
-            <configuration>
-              <skip>${skipDocs}</skip>
-              <backend>html5</backend>
-              <attributes>
-                <source-highlighter>coderay</source-highlighter>
-                <!-- avoid content security policy violations -->
-                <linkcss>true</linkcss>
-                <copycss>true</copycss>
-              </attributes>
-            </configuration>
-          </execution>
-        </executions>
-        <dependencies>
-          <dependency>
-            <groupId>org.asciidoctor</groupId>
-            <artifactId>asciidoctorj</artifactId>
-            <version>${asciidoctorj.version}</version>
-          </dependency>
-        </dependencies>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <version>3.1.0</version>
-        <executions>
-          <execution>
-            <id>assemble</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
-            <configuration>
-              <descriptors>
-                <descriptor>assembly.xml</descriptor>
-              </descriptors>
-              <recompressZippedFiles>true</recompressZippedFiles>
-              <finalName>documentation-${project.version}</finalName>
-              <appendAssemblyId>false</appendAssemblyId>
-              <outputDirectory>target/</outputDirectory>
-              <workDirectory>target/assembly/work</workDirectory>
-              <tarLongFileMode>posix</tarLongFileMode>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
   <profiles>
     <profile>
-      <id>documentation-pdf</id>
+      <id>release</id>
       <activation>
         <property>
           <name>documentation-pdf</name>
         </property>
       </activation>
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>3.1.0</version>
+            <executions>
+              <execution>
+                <id>assemble</id>
```

Comment:
Give a more meaningful name to this execution: what is it assembling exactly? Maybe `assemble-pdf`.

Reply:
We already have `<id>assemble-pdf</id>` phase

Reply:
These are execution ids, not phases, but I get your point, I mixed the two executions. Let's call this one `assemble-html` then.